coresimd/x86_64/mod.rs

```diff
@@ -38,3 +43,64 @@ pub use self::bswap::*;
 
 mod rdrand;
 pub use self::rdrand::*;
+
```

Comment:
I would put the intrinsic in its own module for consistency with the others, but.. the module would just contain one intrinsic..